docs/tutorial/wordpress-source-plugin-tutorial.md

```diff
@@ -157,12 +163,194 @@ export const pageQuery = graphql`
     }
   }
 `
+//highlight-end
 ```
 
 Save these changes and look at localhost:8000 to see your new homepage with list of sorted blog posts!
 
+![WordPress home after query](/images/wordpress-source-plugin-home.jpg)
+
 > **NOTE:** to future editors: it would be useful to also have examples of how to load blog posts to their own individual pages. And helpful to insert a screenshot of the final result here
 
-### Create slugs for each blogpost
+## Create pages for each blog post and link to them
+
+An index page with a post title and excerpt is great, but we should also build pages out for each of the blog posts, and link to them from our `index.js` file.
+
+To do this, we need to:
+
+1. Create pages for each blog post
+2. Link up the title on the index page with the post page.
+
+If you haven't already, please read through [Part 7](/tutorial/part-seven/) of the foundational tutorial, as it goes through the concept and examples of this process.
+
+### Creating pages for each blog post.
+
+In Part 7 of the tutorial, the first step in creating pages in creating slugs for the markdown files. Since we are using WordPress and not Markdown files, we can grab the slugs that get returned from our API call to the WordPress source. We can skip creating slugs, since we already have them.
```

Comment:
```suggestion
In Part 7 of the tutorial, the first step in creating pages is creating slugs for the markdown files. Since we are using WordPress and not Markdown files, you can grab the slugs that get returned from your API call to the WordPress source. You can skip creating slugs, since you already have them.
```